corehq/apps/api/resources/v0_5.py

```diff
@@ -253,7 +253,7 @@ def _update(self, bundle):
                     should_save = True
         return should_save
 
-    def obj_create(self, bundle, request=None, **kwargs):
+    def obj_create(self, bundle, **kwargs):
```

Comment:
can you share why was this incorrect?

Reply:
because tastypie never passes the request to this method - I don't know why it was ever added.